StarWars.Core/ViewModels/CharacterDetailViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using MvvmCross.Commands;
+using MvvmCross.Navigation;
+using MvvmCross.ViewModels;
+using StarWars.Core.Models;
+
+namespace StarWars.Core.ViewModels;
+
+public sealed class CharacterDetailViewModel : MvxViewModel<CharacterModel>
+{
+    private readonly IMvxNavigationService _navigationService;
+    public IMvxAsyncCommand GoBackCommand { get; }
```

Comment:
I'd suggest you to write command as 

private IMvxAsyncCommand _goBackCommand;

public IMvxAsyncCommand GoBackCommand => _goBackCommand ??= new MvxAsyncCommand(GoBack);

It's holywar topic, but in my opinion = it's easier to read and find